torch/_inductor/lowering.py

```diff
@@ -1328,7 +1328,7 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten.count_nonzero)
 make_fallback(aten.cummax)
 make_fallback(aten.cummin)
-make_fallback(aten.cumprod)
+make_fallback(aten.cumprod, warn=False)
```

Comment:
intentional in this PR?

Reply:
It looks intentional since the issue I filed showed this log line:
```
[2023-04-10 12:28:08,991] torch._inductor.utils: [WARNING] make_fallback(aten.cumprod): a decomposition exists, we should switch to it
```

Reply:
Yes.

Reply:
lol now we have 2 lines disabling cumprod warnings